CONTRIBUTING.md

```diff
@@ -1,4 +1,134 @@
 # Contributing to Theatre.js
 
->`@todo` Write the contribution guide.
+## Getting Started
 
+### Prerequisites
+
+Make sure you have [Node](https://nodejs.org/) installed at v14.0.0+
+
+```sh
+node -v
+
+v14.0.0
+
+```
+
+and [Yarn](https://classic.yarnpkg.com/en/) at v2.0.0+.
```

Comment:
I've got 2.4.2 currently installed, so I just set 2.0.0 as a minimum requirement.

Reply:
I think with Yarn, you install 1.x.x globally, and when you `cd` into a package that uses yarn `2.x`, it switches to 2.x automatically. Yarn 2.x's whole bundle is even [committed](https://github.com/AriaMinaei/theatre/blob/69c6aa9af226ca4dc852cad1145b9cd0f42f9647/.yarn/releases/yarn-2.4.2.cjs) to the repo.

Reply:
So we could say something like:

> Install the latest version of yarn 1.x globally (`npm install -g yarn`).